utils/scripts/organization.ts

```diff
@@ -185,4 +185,24 @@ export const addUserToOrgWithEmail = async(email: string, orgID: number) => {
     }
     return null;
 
+}
+
+type OrgAndOrgType = {
+    organizations: {
+        id: number;
```

Comment:
Does it need to return the organization ID? Only isNonProfit is used where it's currently called.